### PR TITLE
registry

### PR DESCRIPTION
**Pull request by jsendo**
new directory structure
local requires
key attributes added to components

### CODE REVIEW

File: components/aws/sources/new-scheduled-tasks/README.md [hunk 1]
```diff
@@ -24,18 +24,19 @@ You can also listen for these events in your own app / infra, by [subscribing to
...
-  - [An AWS IAM Policy for creating the necessary AWS resources](#an-aws-iam-policy-for-creating-the-necessary-aws-resources)
-  - [An AWS IAM user, with an access and secret key](#an-aws-iam-user-with-an-access-and-secret-key)
-- [AWS Resources](#aws-resources)
-  - [Estimated AWS Costs](#estimated-aws-costs)
-- [HTTP API](#http-api)
-  - [Scheduling a task](#scheduling-a-task)
-  - [Cancel scheduled tasks](#cancel-scheduled-tasks)
-- [Processing scheduled tasks](#processing-scheduled-tasks)
-- [Example: Schedule a task 30 seconds in the future](#example-schedule-a-task-30-seconds-in-the-future)
+- [AWS Task Scheduler](#aws-task-scheduler)
```
Comment (dylburger) [inline components/aws/sources/new-scheduled-tasks/README.md:27]: @psavkar the ToC generator always includes the `h1` tag but I remove it because it's not a part of the core content. Mind removing and indenting the other sections to the left?

File: components/dropbox/sources/all-updates/all-updates.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
-const dropbox = require('https://github.com/PipedreamHQ/pipedream/components/dropbox/dropbox.app.js')
+const dropbox = require('../../dropbox.app.js')
 
 module.exports = {
-  name: "Dropbox - All File and Folder Updates",
+  key: "dropbod-new-or-modified-file-or-folder",
```
Comment (dylburger) [inline components/dropbox/sources/all-updates/all-updates.js:4]: s/dropbod/dropbox/

File: components/faunadb/README.md [hunk 3]
```diff
@@ -6,17 +6,18 @@ FaunaDB Event Sources collect data from FaunaDB (for example, changes to a colle
 
 <!--ts-->
 
-- [Sources](#sources)
-  - [Changes to Documents in a Collection](#changes-to-documents-in-a-collection)
-- [Workflows](#workflows)
+- [FaunaDB Event Sources](#faunadb-event-sources)
```
Comment (dylburger) [inline components/faunadb/README.md:9]: Same ToC h1 issue here

File: components/github/sources/new-commit-comment/new-commit-comment.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-const eventTypes = ['created'];
+const github = require("../../github.app.js");
+const eventNames = ["commit_comment"]
+const eventTypes = ['created']
 
 module.exports = {
+  key: "github-new-commit-comment",
   name: "New Commit Comment (Instant)",
-  description: "Triggers when new commit comments are created",
+  description: "Emitan event when a new commit comment is created",
```
Comment (dylburger) [inline components/github/sources/new-commit-comment/new-commit-comment.js:8]: No space between "emit" and "an"

File: components/pipedream/sources/new-scheduled-tasks/README.md [hunk 5]
```diff
@@ -17,11 +17,12 @@ You can also listen for these events in your own app / infra, by [subscribing to
 
 <!--ts-->
 
-- [HTTP API](#http-api)
-  - [Scheduling a task](#scheduling-a-task)
+- [Task Scheduler](#task-scheduler)
```
Comment (dylburger) [inline components/pipedream/sources/new-scheduled-tasks/README.md:20]: Same ToC issue here

File: components/slack/sources/new-message-in-channels/new-message-in-channels.js [hunk 6]
```diff
@@ -1,14 +1,17 @@
-const slack = require('https://github.com/PipedreamHQ/pipedream/components/slack/slack.app.js')
+const slack = require('../../slack.app.js')
 
 module.exports = {
-  name: "Slack - New Message In Conversation(s)",
+  key: "slack-new-message-in-channels",
```
Comment (dylburger) [inline components/slack/sources/new-message-in-channels/new-message-in-channels.js:4]: Maybe keep the "New Message in Conversation" format, since this lets you listen for messages in group chats as well as channels, I believe? May want to confirm with Dan.

File: docs/docs/event-sources/http/README.md [hunk 7]
```diff
@@ -2,10 +2,13 @@
 
 This tutorial will walk you through how to:
 
-1. [Install the Pipedream CLI](#install-the-pipedream-cli)
-2. [Link the CLI to your Pipedream account](#sign-up-for-pipedream)
-3. [Create an HTTP source](#create-an-http-source)
-4. [Consume events from that source](#consume-events-from-your-http-source)
+- [Creating an HTTP source](#creating-an-http-source)
```
Comment (dylburger) [inline docs/docs/event-sources/http/README.md:5]: Same ToC issue here, I'd maybe just revert the change? I think we originally had the numbers there for clarity since it was a set of steps the user was supposed to follow.

File: components/aws/sources/new-scheduled-tasks/README.md (hunk 1 above)
Comment (psavkar) [reply components/aws/sources/new-scheduled-tasks/README.md:27]: done

File: components/faunadb/README.md (hunk 3 above)
Comment (psavkar) [reply components/faunadb/README.md:9]: done

File: components/github/sources/new-commit-comment/new-commit-comment.js (hunk 4 above)
Comment (psavkar) [reply components/github/sources/new-commit-comment/new-commit-comment.js:8]: done

File: components/pipedream/sources/new-scheduled-tasks/README.md (hunk 5 above)
Comment (psavkar) [reply components/pipedream/sources/new-scheduled-tasks/README.md:20]: done

File: docs/docs/event-sources/http/README.md (hunk 7 above)
Comment (psavkar) [reply docs/docs/event-sources/http/README.md:5]: done

File: components/slack/sources/new-message-in-channels/new-message-in-channels.js (hunk 6 above)
Comment (psavkar) [reply components/slack/sources/new-message-in-channels/new-message-in-channels.js:4]: just closing the loop. made a call to leave as it. thanks!
